arrows/ocv/tests/test_image.cxx

```diff
@@ -137,7 +137,7 @@ TYPED_TEST(image_io, type)
   kwiver::vital::image_of<pix_t> img( 200, 300, TypeParam::depth );
   populate_vital_image<pix_t>( img );
 
-  auto const image_path = kwiver::testing::temp_file_name( "test-", ".tiff" );
+  auto const image_path = kwiver::testing::temp_file_name( "test-", ".png" );
```

Comment:
I'm not sure that it matters much, but what was the motivation for this change?

Reply:
Because on my Linux machine, tiff was not saving out RGBA images as RGBA images but as RGB images, thus a test was failing.  I was building with the bootstrap.  The commit can be reverted if desired.

Reply:
That seem like a reasonable reason to change it, I was just curious.

Reply:
Interesting; AFAIK it WFM prior to the change that broke these tests... there must be something funky about libtiff from bootstrap vs. Fedora's package? I don't have an objection to this change, but it might be worth tracking that down, or at least making a note somewhere, since it could cause problems if someone wants to load data from a 4-channel TIFF.